### DISCUSSION TITLE
Increasing Blazorise community

### DISCUSSION THREAD

**Discussion by GevorDanielyan** [Megabit/Blazorise#3876] [Show and tell]
Hello everyone. 

I'm added Blazorise in a collection of awesome .NET libraries, tools, frameworks, and software. Since [awesome-dotnet](https://github.com/quozd/awesome-dotnet) has a fairly large community, adding Blazorise in [there](https://github.com/quozd/awesome-dotnet#blazor), will help developers to know about it and use this awesome component library. 

Best regards,
[Gevorg Danielyan](https://github.com/GevorDanielyan)

**Comment by stsrki** [comment]
Excellent. Thank you for sharing!

**Reply by GevorDanielyan** [reply]
> Excellent. Thank you for sharing!

Thank you too
